src/librustc/session/config.rs

```diff
@@ -1052,7 +1052,7 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     linker_flavor: Option<LinkerFlavor> = (None, parse_linker_flavor, [UNTRACKED],
                                            "Linker flavor"),
     fuel: Option<(String, u64)> = (None, parse_optimization_fuel, [TRACKED],
-        "Set the optimization fuel quota for a crate."),
+        "Set the optimization fuel quota for a crate"),
     print_fuel: Option<String> = (None, parse_opt_string, [TRACKED],
         "Make Rustc print the total optimization fuel used by a crate."),
```

Comment:
What about this period?

Reply:
Didn't notice this one. Should I remove and make them start with lowercase so that everything is the same then?